patches/api/0378-allow-setting-of-arrowpickup.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Aaron <78169748+tl0x@users.noreply.github.com>
+Date: Fri, 8 Apr 2022 17:03:46 -0400
+Subject: [PATCH] allow setting of arrowpickup
+
+
+diff --git a/src/main/java/org/bukkit/entity/Arrow.java b/src/main/java/org/bukkit/entity/Arrow.java
+index 8814519df13c5e9475774f4bce4c8c6a462c2978..7f80a6d1a0f1f2d1889062a1f451e311bdc92e1a 100644
+--- a/src/main/java/org/bukkit/entity/Arrow.java
++++ b/src/main/java/org/bukkit/entity/Arrow.java
+@@ -2,6 +2,7 @@ package org.bukkit.entity;
+ 
+ import java.util.List;
+ import org.bukkit.Color;
++import org.bukkit.inventory.ItemStack;
+ import org.bukkit.potion.PotionData;
+ import org.bukkit.potion.PotionEffect;
+ import org.bukkit.potion.PotionEffectType;
+@@ -94,4 +95,11 @@ public interface Arrow extends AbstractArrow {
+      * in a state with no Custom Effects and PotionType.UNCRAFTABLE
+      */
+     void clearCustomEffects();
++
++    /**
++     * Sets the ItemStack given when the arrow is picked up.
++     *
++     * @param itemStack
++     */
```

Comment:
I would make some mention that this isn't persisted across entity unload/loads. aka its not saved.

Reply:
why not save the itemstack as bytes in it’s pdc?